Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 
 members = [
+    "aries",
```

Comment:
I'd suggest calling the crate `aries-vcx`, there's already `aries` in crates.io anyway

Reply:
`aries_vcx`, so it's easily addressable